src/main/scala/cromwell/binding/values/WdlBoolean.scala

```diff
@@ -0,0 +1,30 @@
+package cromwell.binding.values
+
+import cromwell.binding.WdlExpressionException
+import cromwell.binding.types.WdlBooleanType
+
+case class WdlBoolean(value: Boolean) extends WdlPrimitive {
+  val wdlType = WdlBooleanType
+  override def equals(rhs: WdlValue): WdlBoolean = {
+    rhs match {
+      case r:WdlBoolean => WdlBoolean(value == r.value)
+      case _ => invalid(s"$value || $rhs")
+    }
+  }
+  override def lessThanOrEqual(rhs: WdlValue): WdlBoolean = equals(rhs)
+  override def greaterThanOrEqual(rhs: WdlValue): WdlBoolean = equals(rhs)
```

Comment:
Python has the same semantics as Scala for this particular case.  In Python, True and False are basically just aliases for 1 and 0 (at least when it comes to expression evaluation).  We're not following Python semantics exactly because `1 + true` is an error in WDL (and Scala) but it's 2 in Python.  I'm just guessing we want to keep `Int + Boolean` as an error in WDL, though